test/unit/config/config.test.js

```diff
@@ -164,3 +164,21 @@ tap.test('#publicSettings', (t) => {
     t.end()
   })
 })
+
+tap.test('should handle ESM modules gracefully', (t) => {
```

Comment:
nit: technically this is not a unit test.  it's probably better suited in `test/integration/config` suite.

Reply:
Oh, I thought an integration test would be actually running the whole import of `esm-good/index.js`. I have that half-written too.

Are you saying this isn't unit-y enough because I don't mock out the import of `newrelic.js`?

Reply:
you're setting an env var that requires the relevant config from a new package that was defined in the repo.  So yes technically unit tests would mock the require and that it threw the ERR_REQUIRE_ESM.

Reply:
Okay, changed it to a full-blown integration test.